plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/S3ServiceTests.java

```diff
@@ -40,7 +40,7 @@
 public class S3ServiceTests extends AbstractS3RepositoryTestCase {
     public void testCachedClientsAreReleased() {
         final S3Service s3Service = new S3Service(configPath());
-        final Settings settings = Settings.builder().put("endpoint", "http://first").build();
+        final Settings settings = Settings.builder().put("endpoint", "http://first").put("region", "region").build();
```

Comment:
With v2 we are making the `Region.of` call and sending a `Region` type to the SDK (in contrast to the string that we were passing earlier). It might be a good idea to have some logic to do sanity testing of the region setting (which is a string), otherwise the `Region.of` call will throw a fatal error. We should be able to use OpenSearch settings validators for this. Thoughts?

Reply:
Could it be by design? When a new region is created, would you want to be validating against a fixed list?

Reply:
If we do validate, it should be against `Region.values()` in my opinion.

Reply:
But that becomes invalid with a new region. I think we shouldn't.